keps/prod-readiness/sig-scheduling/1923.yaml

```diff
@@ -0,0 +1,6 @@
+kep-number: 1923
+alpha:
+  approvers:
+    - "@Huang-Wei"
+    - "@ahg-g"
```

Comment:
approvers should be updated, only guys in the alias of "prod-readiness-approvers" could approve this, will update and maybe split this PR later.